LayoutTests/http/tests/navigation/page-cache-getUserMedia-pending-promise.html

```diff
@@ -10,33 +10,33 @@
 jsTestIsAsync = true;
 let restoredFromPageCache = false;
 
-if (window.testRunner)
-    testRunner.setUserMediaPermission(true);
```

Comment:
Why are we dropping this?

Reply:
Because the request will almost always be invalidated (due to navigation) before permission check on UI delegate, so there is no need to enable it. 
The test also does not verify getUserMedia is rejected as there is a chance that TestRunner set the permission enabled by default, and the test will can become flaky if the user permission check is done before invalidation.

Reply:
I didn't understand much of that. CAN we keep this or not? I'm worried we're disabling the test.